src/errno.rs

```diff
@@ -1,3 +1,16 @@
+//! Safe wrappers around errno functions
+//!
+//! # Example
+//! ```
+//! use nix::errno::Errno;
+//!
+//! Errno::set(Errno::EIO);
```

Comment:
Doc needs an update

Reply:
Updated anyway already, but yeah I initially left that intentionally as a matter of preference.